CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [3.0.1] - TBD
+### Added
+- Support for running configuring read-only HMS with Ranger audit-only
```

Comment:
Do you mean 

> Support for configuring read-only HMS with Ranger audit-only

 ?

Reply:
Haha, yes!  That's what I get for doing this on Friday evening. :)